targets/TARGET_ARM_FM/TARGET_FVP_MPS2/TARGET_FVP_MPS2_M0/device/CMSDK_CM0.h

```diff
@@ -270,7 +270,7 @@ typedef struct {
 #define CMSDK_DUALTIMER1_CTRL_INTEN_Msk      (0x1ul << CMSDK_DUALTIMER1_CTRL_INTEN_Pos)       /* CMSDK_DUALTIMER1 CTRL_INTEN: CTRL Int Enable Mask */
 
 #define CMSDK_DUALTIMER1_CTRL_PRESCALE_Pos   2                                                /* CMSDK_DUALTIMER1 CTRL_PRESCALE: CTRL PRESCALE Position */
-#define CMSDK_DUALTIMER1_CTRL_PRESCALE_Msk   (0x3ul << CMSDK_DUALTIMER1_CTRL_PRESCALE_Pos)    /* CMSDK_DUALTIMER1 CTRL_PRESCALE: CTRL PRESCALE Mask */
+#define CMSDK_DUALTIMER1_CTRL_PRESCALE_Msk   (0x1ul << CMSDK_DUALTIMER1_CTRL_PRESCALE_Pos)    /* CMSDK_DUALTIMER1 CTRL_PRESCALE: CTRL PRESCALE Mask */
```

Comment:
This reduces the field size from 2 bits to 1.
What are doing the bits in position 3 and 4 ?

Reply:
bit 4 is an undefined bit, bit 3 is another prescale bit, maybe I should keep the SDK definition as it was. Anyway, I will revert it based on Martin's comments. I will raise another PR to change SDK.

Reply:
Indeed, if the field is 2 bits wide this mask should stay at `0x3ul`.
To set the value in the register you should write
```c
US_TICKER_TIMER1->TimerControl &= ~CMSDK_DUALTIMER1_CTRL_PRESCALE_Msk;
US_TICKER_TIMER1->TimerControl |= 0x1 << CMSDK_DUALTIMER1_CTRL_PRESCALE_Pos; // you can replace 0x1 by any value in [0..3]
```